packages/shell-api/src/shell-internal-state.ts

```diff
@@ -209,8 +209,8 @@ export default class ShellInternalState {
       contextObject.console = {};
     }
     for (const key of ['log', 'warn', 'info', 'error']) {
-      contextObject.console[key] = async(...args: any[]): Promise<void> => {
-        return await contextObject.print(...args);
+      contextObject.console[key] = (...args: any[]): any => {
```

Comment:
```suggestion
      contextObject.console[key] = (...args: any[]): Promise<void> => {
```

would still be okay and accurate, right?

Reply:
Ah yeah, removed that just as a first while testing...